ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -108,10 +108,54 @@ presubmits:
         - --template=/usr/local/e2e-aws-all
         - --target=e2e-aws-all
         env:
-        - name: TEST_FOCUS
```

Comment:
Don't you want focus + skip? If you don't focus conformance you get a _LOT_ of tests as your base to skip from

Reply:
> Don't you want focus + skip? If you don't focus conformance you get a LOT of tests as your base to skip from

Ah, that makes sense.  Restored with 7991c1a -> d486714.